geoportailv3/static/js/draw/featurepopupdirective.js

```diff
@@ -95,6 +126,19 @@ app.FeaturePopupController.prototype.close = function() {
 
 
 /**
+ * @return {boolean} return true if is editable by the user
+ * @export
+ */
+app.FeaturePopupController.prototype.isEditable = function() {
+  if (goog.isDefAndNotNull(this.feature) &&
+      this.feature.get('__source__') == 'mymaps') {
+    return this.appMymaps_.isEditable();
+  }
+  return true;
```

Comment:
I don't know. I was just surprised that the function returned `true` when there was no feature. So I wanted to check this with you.